src/common/ini_handler.c

```diff
@@ -2,9 +2,17 @@
 
 #include "ini_handler.h"
 #include "ff.h"
+#include "eeprom.h"
+#include "lwip/dhcp.h"
+#include "lwip/netifapi.h"
+#include "dns.h"
+#include "ini.h"
 
+#define MAX_UINT16  65535
 static const char network_ini_file_name[] = "/lan_settings.ini"; //change -> change msgboxes in screen_lan_settings
 
+static int ini_config_handler(void *user, const char *section, const char *name, const char *value);
+
 uint8_t ini_save_file(const char *ini_save_str) {
```

Comment:
save ini is not needed at the moment, should be avoided to reduce the complexity in work

Reply:
this function and ini_load_file() is used by screen_lan_settings, it cannot be avoided.